redux/reducers.js

```diff
@@ -0,0 +1,52 @@
+import { UPDATE_TEXT, FILTER_BY_NAME } from "./actions";
+//const data = require("../data.json");
+let data;
+
+async function JSONResponse () {
+
+  let promise = new Promise((resolve, reject) => {
+    var XMLHttpRequest = require('xhr2');
+    let req = new XMLHttpRequest();
+    let currentData = "";
+    req.open("GET", "https://api.jsonbin.io/v3/b/619fe92201558c731cc8d669/latest", true);
+    req.setRequestHeader("X-Master-Key", "$2b$10$uhR.5vVjjd1Oxt5/VemqBemLTV2utVcdwvRBW.9NuByKPccbG67Oa");
```

Comment:
que tal si guardamos esta key en un archivo .env? 
https://stackoverflow.com/questions/64916923/react-native-storing-api-key

Reply:
Couldn't work with the ENV's